### PR TITLE
Edits from sabrina savage

### PR DESCRIPTION
**Pull request by ehsteve**
So many missing commas and now with fewer witches. Thanks @SabrinaSavage!

### CODE REVIEW

**Comment by ehsteve** [comment]
This addresses issue #128.

File: sections/3-support.tex [hunk 1]
```diff
@@ -2,12 +2,12 @@ \section{Support and Sustainability}
 \label{sec:support}
 
 To date, the \sunpyproj relies largely on unpaid, volunteer efforts from early career scientists.
-The project has not received any significant direct financial support for its work facilitating and promoting open source and open development including developing the \sunpypkg package itself.
-This presents a number of challenges which the \astropy community also faces and have been described in \cite{PriceWhelan:2018ji} and \cite{Muna2016}.
+The project has not received any significant direct financial support for its work facilitating and promoting open source and open development. including developing the \sunpypkg package itself.
```
Comment (hayesla) [inline sections/3-support.tex]: should be a comma not a full stop

File: sections/3-support.tex [hunk 2]
```diff
@@ -2,12 +2,12 @@ \section{Support and Sustainability}
...
+This This lack of financial support presents a number of challenges that the \astropy community also faces and have been described in \cite{PriceWhelan:2018ji} and \cite{Muna2016}.
```
Comment (hayesla) [inline sections/3-support.tex]: double this

File: sections/6-coordinates.tex [hunk 3]
```diff
@@ -15,31 +15,31 @@ \subsection{Solar Coordinates}
...
+Support is  provided for querying ephemeris information from a few different sources, including the active ephemeris in \package{astropy.coordinates}, JPL ephemeris, and JPL HORIZONS\footnote{\url{https://ssd.jpl.nasa.gov/?horizon}}.
 
 \begin{figure}
     \gridline{\fig{figures/fig_fieldlines_aia.pdf}{0.3\textwidth}{(a)}
               \fig{figures/fig_venus_transit.pdf}{0.3\textwidth}{(b)}
               \fig{figures/fig_coronagraph_starfield.pdf}{0.3\textwidth}{(c)}
               }
     \caption{Several example use cases of the coordinate machinery provided by the \package{sunpy.coordinates} subpackage.
-    (a) Magnetic field lines traced from a potential field extrapolation overlaid on an SDO/AIA 171 \AA{} AIA observation of an active region from 2019 March 10 00:00:09 UTC.
+    (a) Magnetic field lines traced from a potential field extrapolation overlaid on an SDO/AIA 171 \AA{} observation of an active region from 2019 March 10 00:00:09 UTC.
```
Comment (hayesla) [inline sections/6-coordinates.tex:42]: an -> a

**Review by hayesla — APPROVED** [review]

File: sections/6-coordinates.tex (hunk 3 above)
Comment (ehsteve) [reply sections/6-coordinates.tex:42]: thanks!

File: sections/3-support.tex (hunk 2 above)
Comment (ehsteve) [reply sections/3-support.tex]: thanks!

File: sections/3-support.tex (hunk 1 above)
Comment (ehsteve) [reply sections/3-support.tex]: thanks!

**Pull request merged**